### DISCUSSION TITLE
Add the Align Your Steps scheduler

### DISCUSSION THREAD

**Discussion by gonzalu** [TinyTerra/ComfyUI_tinyterraNodes#115] [Ideas]
Is it possible to add the AYS scheduler? 

or at least make it available as an option externally? 

https://research.nvidia.com/labs/toronto-ai/AlignYourSteps/

Thank you

**Comment by TinyTerra** [comment]
[Added AYS Scheduler to KSamps](https://github.com/TinyTerra/ComfyUI_tinyterraNodes/commit/be22be37a4d1cfacb6bd7d865a95d875839b225e)

**Reply by gonzalu** [reply]
LEGEND!!!!
